airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -241,13 +258,17 @@ def run(
         else:
             return results
 
-    @staticmethod
-    def _make_serializable(result):
-        """Transform the databricks Row objects into JSON-serializable lists."""
-        if isinstance(result, list):
-            return [list(row) for row in result]
-        elif isinstance(result, Row):
-            return list(result)
+    def _make_serializable(self, result):
```

Comment:
Can we have this type hinted?

Reply:
Not a staticmethod anymore because checking self.return_serializable is needed. I added typings